metaflow/plugins/__init__.py

```diff
@@ -38,7 +42,8 @@ def get_plugin_cli():
 # careful with the choice of name though - they become top-level
 # imports from the metaflow package.
 from .conda.conda_flow_decorator import CondaFlowDecorator
-FLOW_DECORATORS = [CondaFlowDecorator]
+from .schedule_decorator import ScheduleDecorator
```

Comment:
Should this be in the AWS directory or is it generic?

Reply:
Yes, you are right. I will move it to the aws/stepfunctions directory.